x-pack/plugins/uptime/server/lib/requests/get_snapshot_counts.ts

```diff
@@ -15,21 +15,25 @@ export interface GetSnapshotCountParams {
   dateRangeStart: string;
   dateRangeEnd: string;
   filters?: string | null;
+  query?: string;
 }
 
 export const getSnapshotCount: UMElasticsearchQueryFn<GetSnapshotCountParams, Snapshot> = async ({
   uptimeEsClient,
   dateRangeStart,
   dateRangeEnd,
   filters,
+  query,
 }): Promise<Snapshot> => {
   const context = new QueryContext(
```

Comment:
Not part of your PR, and I'm honestly not 100% sure if we own this or not, but once the number of parameters starts getting really long like this, I think it's best to convert it to one object parameter, so that you know what each parameter represents and you can omit undefined things. Right now it's hard to understand what each parameter represents.